src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/PeriodicShardSyncStrategy.java

```diff
@@ -0,0 +1,38 @@
+package com.amazonaws.services.kinesis.clientlibrary.lib.worker;
+
+/**
+ * An implementation of ShardSyncStrategy.
+ */
+public class PeriodicShardSyncStrategy implements ShardSyncStrategy {
+
+    public static final String NAME = "PeriodicShardSyncStrategy";
+    private PeriodicShardSyncManager periodicShardSyncManager;
+
+    public PeriodicShardSyncStrategy(PeriodicShardSyncManager periodicShardSyncManager) {
+        this.periodicShardSyncManager = periodicShardSyncManager;
+    }
+
+    @Override public String getName() {
+        return NAME;
+    }
+
+    @Override public TaskResult syncShards() {
```

Comment:
nit: Could you have `Override` in its own line.

Reply:
Changed.